CacheWarmer/HydratorCacheWarmer.php

```diff
@@ -23,6 +23,8 @@
  *
  * In the process of generating hydrators the cache for all the metadata is primed also,
  * since this information is necessary to build the hydrators in the first place.
+ *
+ * @internal
```

Comment:
I decided to make all these internal. Not that it makes much of a difference, but at least it communicates that users shouldn't interact with this directly.